docs/_docs/2. functional.md

```diff
@@ -110,37 +107,54 @@ Each table details the number of test cases that pass and fail for each test fil
         }
     });
 
-    new Chart(document.getElementById('summaryChart'), 
-        {
-          type: 'bar',
-          data: {
-          labels: filteredRows.map(row => row[0]),
-          datasets: [{
-             data: filteredRows.map(row => row[1].score),
-             borderColor: filteredRows.map(row => implData.find(impl => impl.shortName === row[0]).color),
-             backgroundColor: filteredRows.map(row => implData.find(impl => impl.shortName === row[0]).color.replace('rgb', 'rgba').replace(')', ',0.2)')),
-             borderWidth: 1
-           }]
-          },
-          options: {
-            plugins: {
-                title: {
-                    display: true,
-                    text: 'Overall functional score (higher is better)'
+    const chartContainer = document.getElementById('complianceCharts');
+
+    const createChart = (column, title, field, suggestedMin) => {
+        const canvas = document.createElement('canvas');
+        chartContainer.append(canvas);
```

Comment:
minor: Can you append a blank line in between each chart please?  In the performance.md I add a `&nbsp;` between each graph to make them more pleasing to the eye.

Perhaps, even better, would be to add a heading for `#### Overall`, `#### Draft_7`, etc. With each heading having the two graphs underneath it?

Reply:
Done, but unfortunately I had to write more explicit code for this